pkg/query-service/app/http_handler.go

```diff
@@ -351,7 +351,7 @@ func (aH *APIHandler) RegisterRoutes(router *mux.Router) {
 	router.HandleFunc("/api/v1/feedback", OpenAccess(aH.submitFeedback)).Methods(http.MethodPost)
 	// router.HandleFunc("/api/v1/get_percentiles", aH.getApplicationPercentiles).Methods(http.MethodGet)
 	router.HandleFunc("/api/v1/services", ViewAccess(aH.getServices)).Methods(http.MethodPost)
-	router.HandleFunc("/api/v1/services/list", aH.getServicesList).Methods(http.MethodGet)
+	router.HandleFunc("/api/v1/services/list", ViewAccess(aH.getServicesList)).Methods(http.MethodGet)
```

Comment:
We should have checked whether `/list` is being used at frontend before adding auth.
If not used then we should have removed

Reply:
@makeavish From where I can see, it was only used by `install.sh` script.
And referenced in telemetry package's ignored path, which I will create a PR to replace with health endpoint.

Reply:
Ok, then we can remove this endpoint and associated functions